src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -229,13 +229,13 @@
 
 (defn make-network-item
   "This function generates props for quo/category component item"
-  [{:keys [network-name color on-change networks state label-props]}]
+  [{:keys [network-name color on-change networks state label-props normal-checkbox?]}]
```

Comment:
what is normal referring to?

Reply:
or more so, what is a not normal checkbox? 🤔

Reply:
The idea was to always use the same type, so added a type prop with a fallback value.